src/utils/frameworks/react/react.mts

```diff
@@ -8,7 +8,7 @@ export default function (componentName: string, folder: string) {
 			{
 				type: "confirm",
 				name: "typescript",
-				message: "Do you want to use Typescript?",
+				message: "Do you want to use Typescript? (default: Yes)",
```

Comment:
There's no need here to specify de default since the confirm prompt already tells you that when showing the question